client/X11/xf_keyboard.c

```diff
@@ -465,6 +465,26 @@ BOOL xf_keyboard_handle_special_keys(xfContext* xfc, KeySym keysym)
 	{
 		return TRUE;
 	}
+	// Nicholai Mitchko
+	// Add a keyboard binding to allow switching workspaces in FullScreenMode
```

Comment:
Please remove the comment about the author and the "end changes" below. When we wanna know wrote what we have git blame for that.